app/src/main/java/org/jellyfin/androidtv/ui/WatchedBadge.kt

```diff
@@ -0,0 +1,62 @@
+package org.jellyfin.androidtv.ui
+
+import android.content.Context
+import android.view.LayoutInflater
+import android.view.View
+import android.widget.FrameLayout
+import kotlinx.android.synthetic.main.badge_watched.view.*
+import org.jellyfin.androidtv.R
+import org.jellyfin.androidtv.model.itemtypes.PlayableItem
+
+class WatchedBadge(context: Context) : FrameLayout(context) {
```

Comment:
If you specify the primary constructor as `(context: Context, private var item: PlayableItem? = null)` it will be a class property with a value of null by default.